### DISCUSSION TITLE
Feature request: "clearable-readonly" prop for QInput

### DISCUSSION THREAD

**Discussion by iliubinskii** [quasarframework/quasar#12238] [Ideas / Proposals] [answered]
"clearable-readonly" prop should show clear button even if input is readonly.

This is useful for the situation when input is actually editable with prepend button.

E.g.:
![image](https://user-images.githubusercontent.com/70687432/151623610-405b69ce-b8db-4211-9508-b29ca8bcaa7e.png)
In this case, QInput is readonly, but user can edit date using dialog opend with prepend icon.
So, clear button should be shown despite readonly attribute.

**Comment by iliubinskii** [comment, marked as answer]
I found that what I need can be achieved with QField.
This discussion can be closed (I don't see where I can do it myself).

**Reply by metalsadman** [reply]
it's fine to leave the discussion for others to see and take idea from.
